### ISSUE TITLE
Configuring storage to use multiple buckets

### ISSUE THREAD

**Issue by neil-rubens** [aws-amplify/amplify-backend#1859]
### Environment information

```plain text
n/a
```

### Description

Is there a way to configuring storage to use multiple buckets.  Example on the docs shows only a single bucket:

```ts
backend.addOutput({
  storage: {
    aws_region: "<region>",
    bucket_name: "<bucket-name>"
  },
});
```

https://docs.amplify.aws/react/build-a-backend/storage/use-with-custom-s3/#step-2---specify-s3-bucket-in-amplifys-backend-config

**Comment by josefaidt** [comment]
Hey @neil-rubens :wave: thanks for raising this! This is not currently supported, but I've marked this as a feature request to evaluate further!

**Comment by neil-rubens** [comment]
Thank you @josefaidt

**Comment by josefaidt** [comment]
Hey @neil-rubens this is now available as of [`@aws-amplify/backend@1.1.0`](https://github.com/aws-amplify/amplify-backend/releases/tag/%40aws-amplify%2Fbackend%401.1.0). closing 🙂

**Comment by neil-rubens** [comment]
For cross-reference purpose; here is the official docs for this new functionality:
https://docs.amplify.aws/react/build-a-backend/storage/set-up-storage/#configure-additional-storage-buckets

see also:
https://aws.amazon.com/about-aws/whats-new/2024/08/aws-amplify-multiple-bucket-storage/
https://aws.amazon.com/blogs/mobile/add-multiple-storage-buckets-amplify/
